### PR TITLE
API

### PR DESCRIPTION
**Pull request by joshualyngdoh**
# Description
- Connect to the API
- Ran linters check
- Update README

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -1,42 +1,37 @@
...
+export const addBook = createAsyncThunk(ADD, async (book, { dispatch }) => {
+  await fetch(url, { method: 'POST', headers: { 'Content-Type': 'application/json' }, body: JSON.stringify(book) });
+  dispatch({ type: ADD, payload: book });
+});
+
+export const removeBook = createAsyncThunk(REMOVE, async (id, { dispatch }) => {
+  await fetch(`${url}/${id}`, { method: 'DELETE' });
+  dispatch({ type: REMOVE, payload: id });
+});
+
```
Comment (Owusu-Desmond) [inline src/redux/books/booksSlice.js]: - [x] Please make sure that the functionality of the applications is working fine. While I was testing your application as you can see from the video below the `add` and `remove` function to the Bookstore API is not working perfectly as expected. When I add the book and refresh the page I can't see the book there which means the book is not added to the bookstore API. Also when I click on the remove button, the book is not been removed. :relaxed: So please make sure that you fix it. :pray: 

[Screencast from 03-01-2023 01:30:06 PM.webm](https://user-images.githubusercontent.com/92303891/222155062-9e4bc1cd-0162-4c34-91a0-148278ebc097.webm)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @joshualyngdoh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

- [x] Why is it that, there are so many future features you can list under the future feature section of your readme but you are saying none at this moment :smile: 
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joshualyngdoh** [comment]
Thank you for the feedback

**Review by jerryowusu — APPROVED** [review]
Hi @joshualyngdoh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

200w


## Highlights
App working fine :heavy_check_mark: 
Great effort so far :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joshualyngdoh** [comment]
Thank you for the feedback

**Pull request merged**